### DISCUSSION TITLE
MH2024K-16SS: Do you own one and can you try something?

### DISCUSSION THREAD

**Discussion by Makuna** [Makuna/DFMiniMp3#113] [General]
I came across some inconsistencies I need to get some clarity on with the MH2024K-16SS variant, which I do not own. Does anyone have one to try something?  

Basically, turn on the debug (see the below link), run the "PlayMp3" example, capture the debug output and get that to me.  Specially looking for when a track finishing playing notification details.

[DfMiniMp3Debug Feature](https://github.com/Makuna/DFMiniMp3/wiki/API-Reference#debug-features)

**Comment by RobertAngermayr** [comment]
Result with a MH2024K-24SS labeled MP3-TF-16P V3.0.

Sd card contains: 
/mp3/ with 3 files 0001.mp3 - 0003.mp3
/01/ with two files 0001_something.mp3 - 0002_something.mp3
/02/ with 50 files 0001_something.mp3 - 0050_something.mp3
/03/ with 5 files 0001_something.mp3 - 0005_something.mp3
/04/ with two files 0001.mp3 - 0002.mp3

initializing...
OUT  7e ff 06 0c 01 00 00 fe ee ef
IN  7e ff 06 41 00 00 00 fe ba ef
IN  7e ff 06 3f 00 00 02 fe ba ef
SD Card, online
OUT  7e ff 06 46 00 00 00 fe b5 ef
OUT  7e ff 06 46 00 00 00 fe b5 ef
OUT  7e ff 06 46 00 00 00 fe b5 ef
version 0
OUT  7e ff 06 43 00 00 00 fe b8 ef
IN  7e ff 06 43 00 00 1e fe 9a ef
volume 30
OUT  7e ff 06 06 01 00 18 fe dc ef
IN  7e ff 06 41 00 00 00 fe ba ef
OUT  7e ff 06 48 00 00 00 fe b3 ef
IN  7e ff 06 48 00 00 3e fe 75 ef
files 62
starting...
OUT  7e ff 06 12 01 00 01 fe e7 ef
IN  7e ff 06 41 00 00 00 fe ba ef
IN  7e ff 06 3d 00 00 3d fe 81 ef
Play finished for #61
OUT  7e ff 06 12 01 00 01 fe e7 ef
IN  7e ff 06 41 00 00 00 fe ba ef
IN  7e ff 06 3d 00 00 3d fe 81 ef
Play finished for #61
OUT  7e ff 06 12 01 00 01 fe e7 ef
IN  7e ff 06 41 00 00 00 fe ba ef
IN  7e ff 06 3d 00 00 3d fe 81 ef
Play finished for #61
OUT  7e ff 06 12 01 00 01 fe e7 ef
IN  7e ff 06 41 00 00 00 fe ba ef

let me know if you need further infos.
regards
robert

**Reply by Makuna** [reply]
Thanks, it looks normal.
